src/models/metadata.ts

```diff
@@ -13,6 +13,30 @@ import { Timestamp } from './timestamp';
 
 type MetadataType = Root | Timestamp | Snapshot | Targets;
 
+/***
+ * A container for signed TUF metadata.
+ *
+ * Provides methods to convert to and from dictionary, read and write to and
```

Comment:
Should this say json? Assuming this is from python tuf?

```suggestion
 * Provides methods to convert to and from json, read and write to and
```